doc/md/examples/grammar.txt

```diff
@@ -292,7 +292,7 @@
 <dec_nonvar> ::= 
     'let' <pat> '=' <exp>
     'type' <id> ('<' <list(<typ_bind>, ',')> '>')? '=' <typ>
-    <obj_sort> <id>? '='? <obj_body>
+    <obj_sort> <id>? (':' <typ>)? '='? <obj_body>
```

Comment:
In the chapter of `class` definitions, `<obj_body>` is said to contain the `=?`, but it is not consistent all over this document. This needs to be fixed-up.

Reply:
I think it was a "hairline space" 0x200b. The `=?` problem was resolved in d1007d119.